modules/dxp/apps/portal-workflow/portal-workflow-metrics-test/src/testFunctional/tests/WorkflowMetricsPerformanceByStep.testcase

```diff
@@ -0,0 +1,274 @@
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-97667 - Verify that the column Avg. Completion Time is displaying the time taken for N tasks to complete from start to end divided by the number of completed tasks during the selected period"
+	@ignore = "Test Stub"
+	@priority = "5"
+	test CalculateAvgCompletionTimeColumnData {
+
+		// TODO LRQA-67682 CalculateAvgCompletionTimeColumnData pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the user can not insert a future date"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test CannotInsertFutureDate {
+
+		// TODO LRQA-67720 CannotInsertFutureDate pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the card (Performance by Step) has no more than 10 entries displayed"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CardDoesNotDisplayMoreThan10Entries {
+
+		// TODO LRQA-67674 CardDoesNotDisplayMoreThan10Entries pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that an unexpected error state is displayed in (Performance by Step) card"
+	@ignore = "Test Stub"
+	@priority = "2"
+	test CardsDisplayErrorState {
+
+		// TODO LRQA-67680 CardsDisplayErrorState pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the card will update the step name if it is changed in Process Builder"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test CardUpdatesStepNameWhenStepNameIsChangedInProcessBuilder {
+
+		// TODO LRQA-67676 CardUpdatesStepNameWhenStepNameIsChangedInProcessBuilder pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the card (Performance by Steps) works as designed when the portal is set to a non-en-US locale"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test CardWorksAsDesignedWhenPortalIsSetToNonEnUsLocale {
+
+		// TODO LRQA-67673 CardWorksAsDesignedWhenPortalIsSetToNonEnUsLocale pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the default date format for en-US in Custom Range filter is MM/DD/YYYY"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test DefaultDateFormatIsMMDDYYYY {
+
+		// TODO LRQA-67691 DefaultDateFormatIsMMDDYYYY pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that the no completed item state is displaying in Performance by Step card (empty card)"
+	@ignore = "Test Stub"
+	@priority = "3"
+	test EmptyStateIsDisplayedWhenThereIsNoCompletedItem {
+
+		// TODO LRQA-67688 EmptyStateIsDisplayedWhenThereIsNoCompletedItem pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects Last 7 Days time period, the result and filter options displayed in accordance with reference"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test FilterItemsFromLast7Days {
+
+		// TODO LRQA-67687 FilterItemsFromLast7Days pending implementation
+
+	}
+
+	@description = "LPS-100674 - Verify that when the user selects (Last 90 Days) time period the result and filter options displayed in accordance with reference"
```

Comment:
Please, remove the parenthesis from (Last 90 Days).
When, in the test map, you have double quotation marks in the test case description, you can only remove them from the stub description.